bigbite/Jake-Week-2
# Feat/contact page

## Description
Created a contact us page for a Mental Health Awareness Website that includes; a navbar, a footer, introductory title, a 'page container' containing a form section on the right and a contact details section on the left. The form section contains 4 areas of input, Name, Email, Message, T&C's Checkbox and upon criteria meeting validation, the submit button will become accessible. The contact details section contains a paragraph, contact information section, and 3 navigation links containing social media links.


## Change Log



- Added a page container utilising display flex.
- Added form within right side container, including the JS validation for this.
- Added a contact information section utilising text gradients to achieve the desired look for the icons.
- Change the font for the whole website to achieve a more 'professional' look.

## Steps to test
To ensure form validation is correct, the submit button will be disabled until **ALL** input fields meet required criteria, this inlcudes;
- Full name being over 5 characters.
- Email being under 255 characters and containing an '@'.
- Message being between 50 - 500 characters.
- Checkbox being ticked.
Once this criteria is met the submit button will be enabled for the user to submit there form.


## Screenshots/Videos

https://github.com/bigbite/Jake-Week-2/assets/53060167/f4f8bb83-4deb-4b66-bef6-3203974ec298




## Checklist:
